agenta-web/src/lib/helpers/formatters.ts

```diff
@@ -5,17 +5,25 @@ const intlNumber = new Intl.NumberFormat("en-US", {
 const intlCurrency = new Intl.NumberFormat("en-US", {
     style: "currency",
     currency: "USD",
-    maximumFractionDigits: 4,
+    maximumFractionDigits: 6,
 })
 
 export const formatNumber = (value = 0) => {
     return intlNumber.format(value)
 }
 
-export const formatCurrency = (value = 0) => {
+export const formatCurrency = (value: number) => {
     return intlCurrency.format(value)
 }
 
 export const formatLatency = (value = 0) => {
     return `${Math.round(value * 1000)}ms`
 }
+
+export const formatTokenUsage = (value: number) => {
+    if (value === 0) {
```

Comment:
same as above if backend is giving 0 then let's display 0 
if it's null we display "-"